numpy/__init__.pxd

```diff
@@ -350,7 +356,10 @@ cdef extern from "numpy/arrayobject.h":
 
     PyObject *PyArray_BASE(ndarray) nogil  # returns borrowed reference!
     PyArray_Descr *PyArray_DESCR(ndarray) nogil  # returns borrowed reference to dtype!
+    PyArray_Descr *PyArray_DTYPE(ndarray) nogil  # returns borrowed reference to dtype! NP 1.7+ alias for descr.
```

Comment:
Heh, I really don't mind just sticking with DESCR... not like we are going to add aliases for all the functions, but right.